elasticsearch/elasticsearch.d.ts

```diff
@@ -331,6 +333,11 @@ declare module Elasticsearch {
         refresh?: boolean;
     }
 
+    export interface DeleteByQueryParams extends GenericParams {
+        index: string;
```

Comment:
Yeah, I guess it makes sense to split those parameters up. 

Nevertheless, in my Kibana Developer Workbench, the documented fields don't have a complete intersection with the working fields for both api calls. But that's not a DT problem.